suite-native/module-passphrase/src/components/PassphraseScreenHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import { useNavigation } from '@react-navigation/native';
+
+import { IconButton, ScreenHeaderWrapper } from '@suite-native/atoms';
+
+export const PassphraseScreenHeader = () => {
+    const navigation = useNavigation();
+
+    return (
+        <ScreenHeaderWrapper>
+            <IconButton
```

Comment:
The `GoBackIcon` has support for `back` icon via its prop. Could not we use it here?

Reply:
This icon is `X`, not chevron left.

Reply:
yes, GoBackIcon has support for this

Reply:
Ah, sorry, when you said `back` icon, I thought you meant the classic chevron. Fixed, but I assume it will be reverted soon because we'll need to extend the handler so that when passphrase submission is happening, we'll need to handle it somehow when back icon is clicked. [d086834](https://github.com/trezor/trezor-suite/pull/11965/commits/d086834e1a35e2b1e075c3b6a097b786e51bc07a)